src/components/TodoList/TodoList.tsx

```diff
@@ -1 +1,19 @@
-export const TodoList = () => {};
+import React from 'react';
+import { TodoInfo } from '../TodoInfo';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+
+type Props = {
+  todolist: Todo[],
+  userlist: User[],
```

Comment:
```suggestion
  todoList: Todo[],
  userList: User[],
```

don't forget to use camelCase